mkt/developers/tasks.py

```diff
@@ -121,7 +122,7 @@ def run_validator(file_path, url=None):
     # uploaded file is on the local filesystem.
     temp_path = tempfile.mktemp()
     with open(temp_path, 'wb') as local_f:
-        with storage.open(file_path) as remote_f:
+        with private_storage.open(file_path) as remote_f:
```

Comment:
Yeah, I think we (or I) will have to walk the code path and make sure these are right once done. But this one seemed obvious enough to do it now.